src/components/Navbar/Navbar/style.scss

```diff
@@ -0,0 +1,150 @@
+@import '../../../scss/variables';
+@import '../../../scss/base';
+
+$white: #ffffff;
```

Comment:
The color variables come from the import above you wouldn't want to redefine it in this file.